.github/styles/Vocab/Products/accept.txt

```diff
@@ -47,49 +84,112 @@ dev
 Dexalot
 Docusaurus
 dotenv
+drizzleOptions
 ecrecover
+Enums
+env
 EOAs
+ERC
+ERC721s
 ESLint
+Eteng
 Ethereum
+Ethers
+ethtx
+eVoting
+expectRevert
 feeRecipient
+finney
+fuji
```

Comment:
I think all reference to Fuji should be Fuji with first letter upper-cased.

Reply:
Resolved